packages/embark/src/lib/modules/console/index.ts

```diff
@@ -84,6 +84,10 @@ class Console {
   }
 
   private get isEmbarkConsole() {
+    if (this.embark.currentContext && this.embark.currentContext.includes("test")) {
```

Comment:
Actually, it might be better if we explicitly check that we are running in the context of the console:
```
  private get isEmbarkConsole() {
    return this.ipc.connected && this.ipc.isClient() && this.embark.currentContext.includes(constants.contexts.console);
  }
```

Reply:
That makes a lot of sense